qkernel/src/syscalls/sys_proxy.rs

```diff
@@ -173,6 +465,115 @@ pub fn SysProxy(task: &mut Task, args: &SyscallArguments) -> Result<i64> {
             );
             return Ok(ret);
         }
+        ProxyCommand::CudaStreamSynchronize => {
+            let ret = HostSpace::Proxy(
+                cmd,
+                parameters,
+            );
+            return Ok(ret);
+        }
+        ProxyCommand::CudaStreamCreate => {
+            unsafe{
```

Comment:
Please try remove all the new unsafe you added.  And even if unsafe is necessary, it should be in a small scope.